python-spec/src/somacore/query/query.py

```diff
@@ -110,3 +185,378 @@ def __del__(self) -> None:
         sdel = getattr(super(), "__del__", lambda: None)
         sdel()
         self.close()
+
+    # Internals
+
+    def read(
```

Comment:
Recommend prefixing this method with an underscore as it is internals.  I originally omitted the underscore, as I wasn't sure if it was internal or public API

I am already confident that I need to significantly restructure `read()` and how it is used by `to_anndata()` for performance reasons.